xep-0280.xml

```diff
@@ -372,37 +393,50 @@
+      <li>The server SHOULD NOT treat multiple enable requests (without an intermediate disable request) as an error;</li>
+      <li>the server SHOULD simply return an IQ-result (if the protocol is already enabled) or an IQ-error (if the client is not permitted to enable Carbons) for any subsequent requests after the first.</li>
+      <li>Similarly, the server SHOULD NOT treat multiple disable requests (without an intermediate enable request) as an error;</li>
+      <li>the server SHOULD return an IQ-result (if the protocols is already disabled) or an IQ-error (if the client's request failed previously) for any subsequent requests after the first.</li>
+    </ul>
   </section2>
   <section2 topic='Interaction with Chat States' anchor='bizrules-chatstates'>
-    <p>Note that &xep0085; recommends sending chat state
-    notifications as chat type messages, which means that they will be
-    subject to Carbon-copying.  This is intentional.</p>
+    <p><strong>Note: </strong>&xep0085; recommends sending chat state notifications as chat type messages, which means that they will be subject to Carbon-copying.  This is intentional.</p>
     <p>Additionally, there are other considerations for clients that implement Carbons and <cite>XEP-0085</cite>:</p>
     <ul>
       <li>Upon receiving an inbound or outbound &lt;gone/&gt; chat state (as a carbon copy) for a given conversation, the client SHOULD visually indicate the conversation is terminated.</li>
-      <li>In order to prevent unwanted termination of conversations on other resources, clients SHOULD NOT send &lt;gone/&gt; chat states on logout, but instead SHOULD count on the broadcast of unavailable presence to convey the change in attention.</li>
+      <li>In order to prevent unwanted termination of conversations on other resources, clients SHOULD NOT send &lt;gone/&gt; chat states on logout, instead</li>
+      <li>clients SHOULD count on the broadcast of unavailable presence to convey the change in attention.</li>
       <li>Upon receiving an outbound notification of any chat state other than &lt;gone/&gt;, the copied client MAY conclude that the sending client has taken responsibility for the conversation, and make appropriate user interface modifications.  For example, notifications could be suppressed on devices receiving the Carbon copies.</li>
     </ul>
   </section2>
   <section2 topic='Handling of Errors' anchor='bizrules-errors'>
-    <p>When a receiving server attempts to deliver a forked message, and that message bounces with an error for any reason, the receiving server MUST NOT forward that error back to the original sender.  The receiving server SHOULD use the sent element in the bounce to determine that an error is from a forked message.</p>
-    <p>This rule is used to prevent some of the half-failure modes that have been an issue in other prototocols.</p>
```

Comment:
Hm.. funny I didn't see that typo when I mangled the sentence. 
Thanks for pointing it out, I'll add a correction this evening (UTC+1).